clients/admin-ui/src/pages/data-discovery/action-center/index.tsx

```diff
@@ -77,38 +75,6 @@ const ActionCenterPage = () => {
       }))
     : [];
 
-  // TODO: [HJ-337] Add button functionality
-
-  // const handleAdd = (monitorId: string) => {
-  //   console.log("Add report", monitorId);
-  // };
-
-  const getWebsiteMonitorActions = useCallback(
```

Comment:
this was done this way to support the standard AntList component's actions param, but we don't use that anymore at all anyway, so I moved it all in to the MonitorFieldListItem instead.